sdk/webpubsub/Azure.Messaging.WebPubSub/tests/TaskExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Azure.Rest.WebPubSub.Tests
+{
+    public static class TaskExtensions
+    {
+        public static async Task OrTimeout(this Task task, int millisecondsDelay = 5000)
```

Comment:
What does 'OrTimeout' mean?

Reply:
it throws when timout. It is used to fail the unit test if some method takes too long to complete

Reply:
the name is weird, but ok